### PR TITLE
Dark Mode

### PR DESCRIPTION
**Pull request by IDEA-godwin**
#19

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,5 +6,9 @@ node_modules
 .env
 .env.*
 !.env.example
+pnpm-lock.yaml
```
Comment (shazow) [inline .gitignore:9]: Why are we getting rid of the pnpm lockfile? Is it a bad idea to allow build environments to be fully in sync?

File: .gitignore [hunk 2]
```diff
@@ -6,5 +6,9 @@ node_modules
...
+.editorconfig
 vite.config.js.timestamp-*
 vite.config.ts.timestamp-*
+
+/.idea
```
Comment (shazow) [inline .gitignore:14]: FYI you can create a global gitignore (in `~/.config/git/ignore`) for your machine to ignore all of your editors' stuff. :)

**Review by shazow — APPROVED** [review]
This is a great start, thank you!

Generally happy to merge, just one question about the `pnpm-lock.yaml` file in case it wasn't intentional.

As a future PR, should we add checking the `prefers-color-scheme` media query, and also maybe saving the selection via cookie/localstorage?

File: .gitignore (hunk 1 above)
Comment (IDEA-godwin) [reply .gitignore:9]: No, it's not. I've had an issue based on conflict by different version of npm, so its mostly habit to not commit it at this point. Would update the PR to include it.

File: .gitignore (hunk 2 above)
Comment (IDEA-godwin) [reply .gitignore:14]: Didn't know that, thanks would do that and update the PR

**Comment by shazow** [comment]
Fantastic, thank you so much for contributing!

**Pull request merged**